CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
 * Add `platform_filters` support for `PBXBuildPhase`.  
   [Dimitris Koutsogiorgas](https://github.com/dnkoutso)
   [#838](https://github.com/CocoaPods/Xcodeproj/issues/838)
+  
+* Add `platform_filters` support for `PBXTargetDependency`.  
```

Comment:
@mangerlahn you might need to rebase to latest master because these changes above and below this entry have already shipped with 1.21.0 of Xcodeproj.

Please rebase your branch and add it to the enhancements section below the master changelog.

Reply:
My bad, sorry for being late with that PR 😅

Reply:
np! thank you!